brand-monitor-system/backend/app/api/v1/webhooks.py

```
@@ -0,0 +1,50 @@
+from fastapi import APIRouter, HTTPException, Request
+
+from ...core.database import SessionLocal
+from ...models.scrape_job import ScrapeJob
+from ...scrapers.data_processor import process_apify_dataset
+
+router = APIRouter()
+
+
+@router.post("/apify/{client_id}")
+async def apify_webhook(client_id: str, request: Request):
```

Ruff (0.14.8)

Unused function argument: `client_id`

(ARG001)